spring-kafka/src/test/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainerTests.java

```diff
@@ -816,4 +816,270 @@ private void testAckOnErrorWithManualImmediateGuts(String topic, boolean ackOnEr
 		logger.info("Stop ack on error with ManualImmediate ack mode");
 	}
 
+	@Test
+	public void testIsChildRunning() throws Exception {
+		this.logger.info("Start isChildRunning");
+		Map<String, Object> props = KafkaTestUtils.consumerProps("test1", "true", embeddedKafka);
+		AtomicReference<Properties> overrides = new AtomicReference<>();
+		DefaultKafkaConsumerFactory<Integer, String> cf = new DefaultKafkaConsumerFactory<Integer, String>(props) {
+
+			@Override
+			protected Consumer<Integer, String> createKafkaConsumer(String groupId, String clientIdPrefix,
+																	String clientIdSuffixArg, Properties properties) {
```

Comment:
This does not look like OK code formatting?
Can you revise it, please, so it looks nice from review perspective?